packages/lodestar/src/chain/forkChoice/forkChoice.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * @module chain/forkChoice
+ */
+
+import {toHexString} from "@chainsafe/ssz";
+import {BeaconState, Slot} from "@chainsafe/lodestar-types";
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {ForkChoice, ProtoArray} from "@chainsafe/lodestar-fork-choice";
+
+import {computeAnchorCheckpoint} from "../initState";
+import {ChainEventEmitter} from "../emitter";
+import {ForkChoiceStore} from "./store";
+
+export class EventedForkChoice extends ForkChoice {
```

Comment:
maybe call it LodestarForkChoice? I get that you are trying to hint that we extend fork choice with event emitting but it's tied to lodestar event emitting and forkchoice store. Just a suiggestion, not a strong opinion.